make/help.mk

```diff
@@ -50,7 +50,6 @@ default_category_tag_end := </category-default>
 DEFAULT_CATEGORY = General
 
 .PHONY: help
-## Show the help.
```

Comment:
note: The last thing on earth the person will have done to see the help is to run `make help`, so this help output doesn't seem super helpful!